File: delete-Nth/delete-Nth.js
const deleteNth = (arr, n) => {
    const output = [];
    const dict = {};
    for(let i = 0; i < arr.length; i++) {
        
        dict[arr[i]] = 1 + (dict[arr[i]] || 0);
        if(dict[arr[i]] > n) {
            continue;
        } else { output.push(arr[i]); };
    };
    return output;
};

module.exports = { deleteNth };

//https://www.codewars.com/kata/554ca54ffa7d91b236000023/train/javascript

File: python-venv/delete_nth/kata_function/delete_nth.py
def delete_nth(arr, int):
    output = []
    dict = {}

    for i in range(len(arr)):
        if arr[i] in dict:
            dict[arr[i]] += 1
        else:
            dict[arr[i]] = 1

        if dict[arr[i]] > int:
            continue
        else:
            output.append(arr[i])

    return output


#https://www.codewars.com/kata/554ca54ffa7d91b236000023/train/python